code-ready/tray-windows
# Add settings window

### How to test
1. Download the tray executable from CI(https://ci.appveyor.com/api/buildjobs/gwv28usxlruy1nd8/artifacts/bin%2Fcrc-tray.zip)
2. Run the daemon `crc daemon` from powershell (since the setup of the daemon is broken on windows, https://github.com/code-ready/crc/pull/1781 will fix that!)
3. Change configs in the UI and verify from the CLI that its updated correctly.

### Screenshots
![config_adv](https://user-images.githubusercontent.com/8885742/102993276-e067f700-4542-11eb-83cb-c043eb5da275.PNG)
![config_basic](https://user-images.githubusercontent.com/8885742/102993279-e1992400-4542-11eb-83db-622a44a965ae.PNG)
